core/src/main/resources/org/gbif/vocabulary/persistence/mappers/ConceptMapper.xml

```diff
@@ -371,6 +371,9 @@
     SELECT key, value, created_by, created
     FROM concept_hidden_label${table_suffix}
     WHERE concept_key = #{entityKey,jdbcType=BIGINT}
+    <if test="query != null">
+      AND normalized_value LIKE '%' || #{query,jdbcType=VARCHAR} || '%'
```

Comment:
if you are comparing against the normalizaed value you also have to normalize the param